bin/beepmein

```diff
@@ -0,0 +1,281 @@
+#!/bin/bash
+
+# beepmein
+
+# A little wrapper around linux 'at' command, that will beep you at specific time, see --help.
+# On Debian systems make sure 'at' is installed with --no-install-recommends.
+
+# Copyright (C) 2019/2020 brontosaurusrex
+# License: GPLv3 or later.
+
+# prepare config dir
+config="$HOME/.cache/beepmein"
+mkdir -p "$config"
+
+# var
+delay="0"
+
+# softlinks to audio samples
+alarmSample="$config/alarm"     # longer sample
+reminderSample="$config/remind" # shorter sample
+
+# Will kill all children with 'killall beepmein'
+trap 'kill $(jobs -p) >/dev/null 2>&1' EXIT
+
+# required
+command -v at >/dev/null 2>&1 || { >&2 echo "I need at: sudo apt install --no-install-recommends at" ; exit 1; }
```

Comment:
When writing specifically for Bash, you can use `&>/dev/null`.

Reply:
Perhaps wrap dependency checking into a function instead of repeating the same construct.